pandas_gbq/gbq.py

```diff
@@ -926,28 +749,52 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         be created in current dir. You can also set PANDAS_GBQ_CREDENTIALS_FILE
         environment variable so as to define a specific path to store this
         credential (eg. /etc/keys/bigquery.dat).
-
         .. [local webserver flow]
             http://google-auth-oauthlib.readthedocs.io/en/latest/reference/google_auth_oauthlib.flow.html#google_auth_oauthlib.flow.InstalledAppFlow.run_local_server
         .. [console flow]
             http://google-auth-oauthlib.readthedocs.io/en/latest/reference/google_auth_oauthlib.flow.html#google_auth_oauthlib.flow.InstalledAppFlow.run_console
         .. versionadded:: 0.2.0
-
     dialect : {'legacy', 'standard'}, default 'legacy'
         'legacy' : Use BigQuery's legacy SQL dialect.
         'standard' : Use BigQuery's standard SQL (beta), which is
         compliant with the SQL 2011 standard. For more information
         see `BigQuery SQL Reference
-        <https://cloud.google.com/bigquery/sql-reference/>`__
-
-    **kwargs : Arbitrary keyword arguments
-        configuration (dict): query config parameters for job processing.
-        For example:
-
-            configuration = {'query': {'useQueryCache': False}}
-
-        For more information see `BigQuery SQL Reference
-        <https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query>`__
+        <https://cloud.google.com/bigquery/sql-reference/>`
+    credentials: credentials object, default None (optional)
+        If generating credentials on your own, pass in. Otherwise, will attempt
+        to generate automatically
+    return_type: {'schema','list','df'}, default 'df'
+        schema returns an array of SchemaField objects, which you can access
+            `from pprint import pprint
```

Comment:
this is *very* confusing to have a return_type parameter. usually you should have separate functions for this.

Reply:
Was originally meant to handle the verify schema test, but I've re-written so no need for return type anymore. You can access the schema by using the `run_query` function now.